jujugui/static/gui/src/app/components/boolean-config/boolean-config.js

```diff
@@ -92,8 +92,8 @@ YUI.add('boolean-config', function() {
               <div className="boolean-config--handle"></div>
             </label>
           </div>
-          <div className="boolean-config--description">
-            {this.props.option.description}
+          <div className="boolean-config--description"
+            dangerouslySetInnerHTML={{__html: this.props.option.description}}>
```

Comment:
After putting some thought into the best approach for this here is what I've come up with:

1. Take the description and split on the links ie) <a></a>.
2. Assign each section to an array but instead of adding the strings for the <a>'s, instead, add a jsx <a> using the values provided in the description (see the regex from below).
3. Add this array to the content of the description div.